packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -48,6 +48,7 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
 
   &__input {
     @include mdc-theme-prop(color, text-primary-on-light);
+    @include mdc-typography-base;
```

Comment:
This should work. What I'm thinking though is we only include `mdc-typography-base` on this selector pattern and not in `.mdc-textfield` as well.

So the work here would be removing the following from `.mdc-textfield`:
```css
@include mdc-typography-base;
@each $prop in (font-size, letter-spacing) {
  #{prop}: map-get(map-get($mdc-typography-styles, subheading2), $prop);
}
```

and putting it in `&__input`'s block instead